### PR TITLE
Study group 12

### PR DESCRIPTION
**Pull request by AntonOsika**
This does two things:
* Create study group 12
* Rename old study groups (according to jekyll date convention) to make them all appear in ordered

### CODE REVIEW

File: _data/layout.yml [hunk 1]
```diff
@@ -1 +1 @@
-promotion: study-group_11
+promotion: study-group_12
```
Comment (Jim-Holmstroem) [inline _data/layout.yml]: oh wait, will this show the correct promotion?

Comment (AntonOsika) [reply _data/layout.yml]: Yes it works – taken from the last line of the study-group.md file I guess:

```
---
title: "Study Group #12: applied NLP and non-verbal robots"
location: "World Trade Center Stockholm, Kungsbron 1"
time: "18:00-20:00, January 16"
speakers:
  - per-klaar
  - taras-kucherenko
organizers:
  - anton-osika
---
`study-group_12` content.
```

Comment (AntonOsika) [reply _data/layout.yml]: image

**Comment by AntonOsika** [comment]
The dimension changes are intentional! (Taras sent me a new picture, and it looks good even though not perfectly square)

**Pull request merged**